src/pages/index.tsx

```diff
@@ -1,3 +1,23 @@
+import React, { useEffect } from 'react';
+import { useHistory } from '@docusaurus/router';
+
+export default function Home() {
+  const history = useHistory();
+
+  useEffect(() => {
```

Comment:
Good thinking! 
Usually, I'm not a big fan of this kind of JS tricks, but it's quite clean and makes the most sense in this context, I can't think of any other solution.